addon/data/selector-ui.js

```diff
@@ -159,6 +159,72 @@ const ui = (function () { // eslint-disable-line no-unused-vars
     boxBottomEl: null
   };
 
+  /** Displays the box which allows the user to choose between selecting a region,
+      archiving the page, or going to my shots.
+  */
+  exports.SelectMode = {
+    display: function (callbacks) {
+      if (document.querySelector(".pageshot-preview-overlay") !== null) {
+        return;
```

Comment:
It prevents opening the mode selector again when already making a selection with the resize handles. It would be better to do it at a higher level and not even allow the creation of the shot context, but I tried a few times and couldn't figure out how to correctly do it that way.